Resources/views/ContentType/view_content_type.html.twig

```diff
@@ -34,102 +34,123 @@
             </p>
 
             <section>
-                <h2 class="ez-contenttype-title">{{ 'content_type.metadata.title'|trans|desc("Global properties")}}</h2>
-                <div class="ez-contenttype-metadata">
-                    <div class="ez-contenttype-metadata-name">{{ "content_type.name"|trans|desc("Name") }}</div>
-                    <div class="ez-contenttype-metadata-value">{{ content_type_name }}</div>
-                </div>
-                <div class="ez-contenttype-metadata">
-                    <div class="ez-contenttype-metadata-name">{{ "content_type.identifier"|trans|desc("Identifier") }}</div>
-                    <div class="ez-contenttype-metadata-value">{{ content_type.identifier }}</div>
-                </div>
-                <div class="ez-contenttype-metadata">
-                    <div class="ez-contenttype-metadata-name">{{ "content_type.description"|trans|desc("Description") }}</div>
-                    <div class="ez-contenttype-metadata-value">{{ content_type.descriptions[language_code]|default('') }}</div>
-                </div>
-                <div class="ez-contenttype-metadata">
-                    <div class="ez-contenttype-metadata-name">{{ "content_type.name_schema"|trans|desc("Content name schema") }}</div>
-                    <div class="ez-contenttype-metadata-value">{{ content_type.nameSchema }}</div>
-                </div>
-                <div class="ez-contenttype-metadata">
-                    <div class="ez-contenttype-metadata-name">{{ "content_type.url_alias_schema"|trans|desc("URL alias name pattern") }}</div>
-                    <div class="ez-contenttype-metadata-value">{{ content_type.urlAliasSchema }}</div>
-                </div>
-                <div class="ez-contenttype-metadata">
-                    <div class="ez-contenttype-metadata-name">{{ "content_type.container"|trans|desc("Container") }}</div>
-                    <div class="ez-contenttype-metadata-value">{{ 'yes_no'|transchoice(content_type.isContainer) }}</div>
-                </div>
-                <div class="ez-contenttype-metadata">
-                    <div class="ez-contenttype-metadata-name">{{ "content_type.default_children_sorting"|trans|desc("Default property for sorting chilren") }}</div>
-                    <div class="ez-contenttype-metadata-value">
-                        {{ ("content_type.sort_field." ~ content_type.defaultSortField)|trans(domain="ezrepoforms_content_type") }} / {{ ("content_type.sort_order." ~ content_type.defaultSortOrder)|trans(domain="ezrepoforms_content_type") }}
+                <details open="open">
```

Comment:
instead of adding `details` here, I would replace the previous `section` element by `details` (and the same should be possible for others `details` as far as I can tell)